Userland/Libraries/LibSQL/AST/Token.h

```diff
@@ -226,15 +228,15 @@ class Token {
     TokenType type() const { return m_type; }
     TokenCategory category() const { return category(m_type); }
 
-    StringView value() const { return m_value; }
+    String const& value() const { return m_value; }
```

Comment:
GitHub isn't letting me comment on the constructor above, but continuing the comment on `Token`'s constructor, it can now be:
```c++
    Token(TokenType type, String value, size_t line_number, size_t line_column)
        : m_type(type)
        , m_value(move(value))
        , m_line_number(line_number)
        , m_line_column(line_column)
    {
    }
```
(note the value being `move`d as well)

Reply:
OK